server/protocols/webadmin/webadmin-mailrepository/src/test/java/org/apache/james/webadmin/routes/MailRepositoriesRoutesTest.java

```diff
@@ -912,7 +920,7 @@ public void clearTaskShouldHaveDetails() throws Exception {
 
     @Test
     public void clearTaskShouldRemoveAllTheMailsFromTheMailRepository() throws Exception {
-        when(mailRepositoryStore.get(URL_MY_REPO)).thenReturn(Optional.of(mailRepository));
+        when(mailRepositoryStore.getByPath(PATH_MY_REPO)).thenAnswer(returnMailRepository);
```

Comment:
Why did you switch `thenReturn` into a less specific `thenAnswer` ?

Reply:
because `theReturn` answer always the same value reference (the `Stream`, which can be consumed only once).